packages/wasm-dpp/test/integration/identity/stateTransition/assetLockProof/validateAssetLockTransactionFactory.spec.js

```diff
@@ -157,27 +144,27 @@ describe('validateAssetLockTransactionFactory', () => {
       executionContext,
     );
 
-    expectValidationError(result, IdentityAssetLockTransactionOutPointAlreadyExistsError);
+    await expectValidationError(result, IdentityAssetLockTransactionOutPointAlreadyExistsError);
 
     const [error] = result.getErrors();
 
     expect(error.getCode()).to.equal(1033);
 
     const transaction = new Transaction(rawTransaction);
 
-    expect(Buffer.isBuffer(error.getTransactionId())).to.be.true();
     expect(error.getTransactionId()).to.deep.equal(Buffer.from(transaction.id, 'hex'));
     expect(error.getOutputIndex()).to.deep.equal(outputIndex);
 
     expect(result.getData()).to.be.undefined();
     expect(stateRepositoryMock.isAssetLockTransactionOutPointAlreadyUsed)
       .to.be.calledOnceWithExactly(
-        transaction.getOutPointBuffer(outputIndex),
-        executionContext,
+        this.sinonSandbox.match((val) => Buffer.from(val)
+          .equals(transaction.getOutPointBuffer(outputIndex))),
+        this.sinonSandbox.match.instanceOf(StateTransitionExecutionContext),
       );
   });
 
-  it('should return valid result', async () => {
+  it('should return valid result', async function () {
```

Comment:
It looks like eslint isn't happy about this line :)

Reply:
It's a warning about unnamed `function`